puppet/modules/profiles/manifests/redmine.pp

```diff
@@ -5,11 +5,14 @@
   # TODO include redmine
   $backup_path = '/var/lib/redmine'
 
-  include profiles::backup::sender
+  class {'profiles::backup::sender':
```

Comment:
We have `restic::user_homedir`:
https://github.com/theforeman/foreman-infra/blob/23157ca170975c0be21046473e02be6236cf64af/puppet/modules/profiles/manifests/backup/sender.pp#L19

I wonder if we set that one correctly. Makes me question if we want the process to also have access to the the SSH key. If the service is compromised they can also easily destroy all backups.

Thinking out loud: could we use https://systemd.io/CREDENTIALS/ to let systemd provide the key in $CREDENTIALS_DIRECTORY? You can override some locations with env vars: https://restic.readthedocs.io/en/stable/040_backup.html#environment-variables

https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html#sftp is not very clear in an easy way to override the key location via env vars.

Reply:
my private ssh config is sprinkled with `IdentityFile ~/.ssh/backup_key`, so I guess the answer is "no"